biogrid/apps/biogrid-mvp/src/assets/icons/house.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
If we can, can we include a screenshot of what the svg looks like in a comment at the top of each file?